README.md

```diff
@@ -10,4 +10,32 @@ bazel test //...
 
 ## Developing Entangled 
 - Be sure to run `./hooks/autohook.sh install` after initial checkout!
-- Pass `-c dbg` for building with debug symbols.
\ No newline at end of file
+- Pass `-c dbg` for building with debug symbols.
+
+### Tools needed in git hook
```

Comment:
Tools required for running git  commit hook

Reply:
thanks!